common/telemetry/config.go

```diff
@@ -43,6 +44,8 @@ import (
 )
 
 const (
+	debugModeEnvVar = "TEMPORAL_OTEL_DEBUG"
```

Comment:
I haven't found/am not aware of a standardized way to change the OpenTelemetry level of detail. By default, OTEL is either on or off.